core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -531,6 +532,16 @@ public ServerBuilder idleTimeoutMillis(long idleTimeoutMillis) {
         return idleTimeout(Duration.ofMillis(idleTimeoutMillis));
     }
 
+    /**
+     * Sets whether to prevent connection going idle when a PING frame is received.
+     *
+     * @param keepAliveOnPing whether to reset idle timeout on PING frame or not.
+     */
+    public ServerBuilder keepAliveOnPing(boolean keepAliveOnPing) {
```

Comment:
How about removing this method and adding this parameter to `pingIntervalMillis` and `pingInterval` methods as the original issue suggested?